### PR TITLE
FungibleToken: improve error messages for non-registered accounts

### PR DESCRIPTION
**Pull request by sept-en**
Currently, we are receiving just _"The account is not registered"_ error when there's no registered account and these functions of FT are called:
* `FungibleToken::internal_deposit()`
* `FungibleToken::internal_withdraw()`
* `StorageManagement::storage_deposit()`
* `StorageManagement::storage::withdraw()`

This is not very convenient and could be sometimes not that easy to find out (especially having complex contracts with cross-contracts calls) which account doesn't exist indeed. We recently ran into this problem when writing some sim tests and spent a lot amount of time trying to track the issue. Making these messages more descriptive by specifying the account id solves the issue.

### CODE REVIEW

File: near-contract-standards/src/fungible_token/storage_impl.rs [hunk 1]
```diff
@@ -23,7 +23,7 @@ impl FungibleToken {
                 env::panic(b"Can't unregister the account with the positive balance without force")
             }
         } else {
-            env::log(b"The account is not registered");
+            env::log(format!("The account {} is not registered", &account_id).as_bytes());
```
Comment (willemneal) [inline near-contract-standards/src/fungible_token/storage_impl.rs:26]:
```suggestion
            log!("The account {} is not registered", &account_id);
```

File: near-contract-standards/src/fungible_token/storage_impl.rs [hunk 2]
```diff
@@ -80,7 +80,7 @@ impl StorageManagement for FungibleToken {
                 _ => storage_balance,
             }
         } else {
-            env::panic(b"The account is not registered");
+            env::panic(format!("The account {} is not registered", &env::predecessor_account_id()).as_bytes());
```
Comment (willemneal) [inline near-contract-standards/src/fungible_token/storage_impl.rs:83]:
```suggestion
            log!("The account {} is not registered", &env::predecessor_account_id());
```

Comment (mfornet) [reply near-contract-standards/src/fungible_token/storage_impl.rs:83]: This should not be `log!` but `env::panic`

File: near-contract-standards/src/fungible_token/storage_impl.rs (hunk 1 above)
Comment (sept-en) [reply near-contract-standards/src/fungible_token/storage_impl.rs:26]: Thanks, this looks better!

File: near-contract-standards/src/fungible_token/core_impl.rs [hunk 3]
```diff
@@ -72,7 +72,7 @@ impl FungibleToken {
     }
 
     pub fn internal_deposit(&mut self, account_id: &AccountId, amount: Balance) {
-        let balance = self.accounts.get(&account_id).expect("The account is not registered");
+        let balance = self.accounts.get(&account_id).expect(format!("The account {} is not registered", &account_id).as_str());
```
[the remark this answers is not in the document]
Comment (evgenykuzyakov) [reply near-contract-standards/src/fungible_token/core_impl.rs:75]: This does run `format!` even when expect is not called. Maybe we should encapsulate this into `internal_balance_of`, so you don't have to run `format!` in the common case when the balance is present

**Pull request merged**